src/keychain/cms.ts

```diff
@@ -124,22 +124,22 @@ export class CMS {
     if (r == null) {
       // @ts-expect-error cms types not defined
       const missingKeys: string[] = recipients.map(r => r.keyId)
-      throw errCode(new Error(`Decryption needs one of the key(s): ${missingKeys.join(', ')}`), codes.ERR_MISSING_KEYS, {
+      throw new CodeError(`Decryption needs one of the key(s): ${missingKeys.join(', ')}`, codes.ERR_MISSING_KEYS, {
         missingKeys
       })
```

Comment:
This is the only piece of code affected by the `props` change in CodeError across all of the libp2p packages.